### PR TITLE
Combitool/HealthScanner Loadout item restrictions

### PR DESCRIPTION
**Pull request by mikomyazaki**
- Restricts two augments (toolset and health-analyzer) on the loadout menu behind appropriate jobs. 
- Removes the health-analyzer and drug dispenser from the exo-stellar exosuit.

Health-analyzers especially are non-trivial to obtain for non-medical jobs, and aren't an appropriate loadout item for everyone to be able to spawn with, in my opinion. I don't think the higher than usual loadout point cost has been enough to prevent these being used for powergaming.

I have a WIP PR that will follow this one that adds the various augments to the roboticist's printer so they can give them to people IC.

### CODE REVIEW

**Comment by mikomyazaki** [comment]
!review

**Comment by FayeLovesU** [comment]
And what jobs are "appropriate" to have a health analyzer?

**Comment by mikomyazaki** [comment]
> And what jobs are "appropriate" to have a health analyzer?

You can [see from the PR](https://github.com/Aurorastation/Aurora.3/blob/9659dc1a013032bada03f4a175eb5b145cf4b428/code/modules/client/preference_setup/loadout/loadout_augments.dm#L77) which jobs I have restricted it to.

**Comment by Roostercat12** [comment]
I don't see how the Med analyzer could be used for powergaming. iirc the only thing they can scan is the person they are installed in. If they are just going to be attached to medical or certain jobs, they may as well be removed.

**Review by alsoandanswer — APPROVED** [review]
Code looks fine.

Personally, I find it absolutely reasonable for it to be restricted. Any loadout items giving any gameplay benefit other than cosmetic is already a stretch.

**Comment by MortalRed** [comment]
To be honest, the idea of an augment just vanishing right out of existence by going visitor or something feels real gamey to me. Clothes I can get, but something that's an actual part of yourself?

**Comment by Gr33d-y** [comment]
Not only do I agree with red, but Ive always seen the medical scanner as a scanner for NON-medical personnel. Security for instance could use it to see if they are ok after getting shot. Medical on the other hand has a scanner in their PDA and will likely never use the augment. not to mention the RP implications of playing an older or sickly character that needs the aug to see if they are ok.
Furthermore, the combitool could be used by other personnel to have more flavor. I personally use one as a first responder to open fire doors to help people out, or as a miner in emergency situations. Restricting the combitool to only be useable if that person has a robotic hand or expanding the jobs that can have it might make more sense thematically than restricting it to just the engineering jobs.

**Comment by Roostercat12** [comment]
recommend making a feedback thread since this one is starting to get clogged with discussion when its for code.

**Comment by mikomyazaki** [comment]
> recommend making a feedback thread since this one is starting to get clogged with discussion when its for code.

I intend to make one (as I said on Discord) after maintainers get back to me about a couple of questions relating to this.

Would prefer to keep the PR comments code-related.

**Review by Arrow768 — APPROVED** [review]

**Comment by mikomyazaki** [comment]
I've asked a couple of times about feedback threads in the developer channel. Maybe maintainers can let me know here before they merge. Thanks.

**Review by Arrow768 — CHANGES_REQUESTED** [review]
As discussed in #developers: 

The loadout option should be available independently of the selected job for consistency reasons. (It is strange if characters with a augment would show up the next day without that just because they are visitors)

Instead a system should be implemented which disables the augments if the character joins with a "invalid" job.
With that the augment is still present (solving the consistency issue), but can not be used.

**Pull request closed without merging**
